shell-ui/src/index.js

```diff
@@ -23,7 +28,7 @@ export type SolutionsNavbarProps = {
   'response-type'?: string,
   'redirect-url'?: string,
   'config-url'?: string,
-  options?: Options,
+  options?: string,
```

Comment:
Maybe a stupid question: why are we moving from a typed object to some free-form string? Is it supposed to be easier when working with WebComponents?

Reply:
WebComponents attributes can sadly not accept object as attributes because the resulting xhtml markup would then be invalid. A common trick is to serialize the object as JSON to enable passing objects to web components 😢

Reply:
Oh :confused: OK